src/EdozoLogo2/EdozoLogo2.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ProductName, StyledLogo } from './EdozoLogo2.styles';
+
+export interface Props {
+  appName?: string;
+  colour?: 'Maps' | 'Occupiers' | 'Insight';
```

Comment:
Not sure we need both of these? if we have `appNaame?: 'Maps' | 'Occupiers' | 'Insight';` could we not infer the colours from there.

Reply:
This would also lock down what the appName can be.